app/services/PreviousYearPeriod.scala

```diff
@@ -34,6 +34,17 @@ trait PreviousYearPeriod {
     }
   }
 
+  def cylbCutoff(frequency: PaymentFrequency, periods: Seq[PeriodWithPaymentDate]): LocalDate = {
+    val periodWithPaymentDate = periods.head
+
+    val cylbDuration = CylbDuration(frequency, periodWithPaymentDate.period)
+
+    (cylbDuration.previousPeriodDays, cylbDuration.equivalentPeriodDays) match {
+      case (0, _) => lastYear(frequency, periodWithPaymentDate.period.period.start).plusDays(1)
+      case _      => calculateDatesForPreviousYear(frequency, periodWithPaymentDate.period.period.start).head.plusDays(2)
```

Comment:
We should try to avoid using `head`. Is there a possibility the list returned will be empty?

Reply:
returned will never be empty, it is either Seq with 1 element or 2 elements

Reply:
@jordanrowe We should model it as a `NonEmptyList`, but that's fine then.